drivers/adc/ad7124/ad7124.c

```diff
@@ -420,7 +420,8 @@ uint8_t ad7124_compute_crc8(uint8_t * p_buf, uint8_t buf_size)
 
 	while(buf_size) {
 		for(i = 0x80; i != 0; i >>= 1) {
-			if(((crc & 0x80) != 0) != ((*p_buf & i) != 0)) { /* MSB of CRC register XOR input Bit from Data */
+			if(((crc & 0x80) != 0) != ((*p_buf & i) !=
+					                   0)) { /* MSB of CRC register XOR input Bit from Data */
```

Comment:
nitpick; this is isn't the nicest wai to break this line;
admittedly, it's also not the nicest code either [from our side]

one way to do it is to use temp vars;
let's say:
```
cmp1 = (crc & 0x80) != 0);
cmp2 = (*p_buf & i);

if (cmp1 != cmp2) {
```

Reply:
not a problem, can edit code as suggested.